packages/devtools_app/lib/src/screens/memory/panes/allocation_profile/allocation_profile_table_view.dart

```diff
@@ -354,15 +330,19 @@ class _AllocationProfileTableControls extends StatelessWidget {
           allocationProfileController: allocationProfileController,
         ),
         const SizedBox(width: denseSpacing),
-        OutlinedIconButton(
-          key: refreshKey,
-          icon: Icons.refresh,
-          onPressed: () => allocationProfileController.refresh(),
+        RefreshButton.icon(
```

Comment:
FYI @CoderDake I think this is something you can use in your inspector pub root directories PR